Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.py

```diff
@@ -0,0 +1,535 @@
+
+        submit_json: Dict[str, Any] = {}
+        try:
+            submit_json = submit_res.json()
+        except Exception:
+            pass
+
+        task_id = _extract_task_id(submit_json) or ""
+        ui_url = f"{base_url.rstrip('/')}/SecurityTasks/Details?taskID={task_id}" if task_id else ""
+
+        # Optional polling (if TaskId exists)
+        if wait_seconds > 0 and task_id:
+            end = time.time() + wait_seconds
+            while time.time() < end:
+                time.sleep(3)
+                chk = check_task_with_api_key(s, base_url, api_key, str(task_id))
+                if chk and str(_ci_get(_ensure_json_like(chk).get('data', chk), 'Status', 'status')).lower() in ('completed', 'done', 'finished'):
+                    parsed = _parse_task_fields(chk, task_id, base_url)
+                    _maybe_set_incident_fields(parsed)
+                    return_results(CommandResults(
+                        outputs_prefix='ShadowxSOCAI.TaskResult',
+                        outputs=parsed,
+                        readable_output=f"ShadowX Task Completed\nURL: {parsed.get('TaskURL', '')}",
+                        raw_response=chk
+                    ))
+                    return
+
+        # Return submit info (include URL if we have it)
+        out = submit_json or {"status": "Submitted"}
+        if isinstance(out, dict) and ui_url:
+            out['TaskURL'] = ui_url
+            if task_id and 'TaskId' not in out:
+                out['TaskId'] = task_id
+
+        return_results(CommandResults(
+            outputs_prefix='ShadowxSOCAI.TaskSubmit',
+            outputs=out,
+            readable_output=f"ShadowX task submitted (API key mode)\nURL: {ui_url}" if ui_url else 'ShadowX task submitted (API key mode)',
+            raw_response=out
+        ))
+        return
+
+    # Cookie/HTML mode
+    email = demisto.params().get('user_email') or ''
+    password = demisto.params().get('user_password') or ''
+    _require_param('user_email', email)
+    _require_param('user_password', password)
+
+    _, effective_base = login_to_dotnet_api(s, base_url, email, password)
+    req_token = get_verification_token(s, effective_base)
+
+    submit_res = create_security_task_cookie(s, effective_base, req_token, payload)
+    if submit_res.status_code != 200:
+        return_error(f"Task submit failed (cookie mode): {submit_res.status_code} {_debug_slice(submit_res.text, 600)}")
+
+    submit_json: Dict[str, Any] = {}
+    try:
+        submit_json = submit_res.json()
+    except Exception:
+        pass
+
+    task_id = _extract_task_id(submit_json) or submit_json.get('taskId') or submit_json.get('taskID') or ''
+    ui_url = f"{effective_base.rstrip('/')}/SecurityTasks/Details?taskID={task_id}" if task_id else ''
+
+    if wait_seconds > 0 and task_id:
+        end = time.time() + wait_seconds
+        while time.time() < end:
```

Comment:
This polling loop for cookie mode appears to be incomplete. It sleeps but doesn't contain any logic to check the task status. This means if `wait_seconds` is used in cookie mode, the command will simply pause and not return the final results as intended.